tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -162,20 +176,11 @@ def pre_build_jobspecs(self):
     return []
 
   def build_jobspec(self):
-    if self.platform == 'windows':
```

Comment:
In def() targets, there is no definition for `RubyArtifact('windows', ...)` so the if self.platform == 'windows' branch will never be run anyway. Leaving as is.